Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs

```diff
@@ -683,11 +709,12 @@ private void HandleOneHandMoveStarted()
             // Calculate relative transform from object to hand.
             Quaternion worldToPalmRotation = Quaternion.Inverse(pointer.Rotation);
             objectToHandRotation = worldToPalmRotation * hostTransform.rotation;
-            objectToHandTranslation = worldToPalmRotation * (hostTransform.position - pointer.Position);
+
+            MixedRealityPose pointerPose = new MixedRealityPose(pointer.Position, pointer.Rotation);
+            MixedRealityPose hostPose = new MixedRealityPose(hostTransform.position, hostTransform.rotation);
+            moveLogic.Setup(pointerPose, pointerData.GrabPoint, hostPose, hostTransform.localScale);
 
             Vector3 worldGrabPoint = pointerData.GrabPoint;
-            // Compute the grab point in local space of the host
-            hostLocalGrabPoint = Quaternion.Inverse(hostTransform.rotation) * (worldGrabPoint - hostTransform.position);
```

Comment:
can we remove hostLocalGrabPoint variable if we are not initializing it? Or, is it already removed?

Reply:
It's already removed:
![image](https://user-images.githubusercontent.com/47415945/61864739-dc592500-aec9-11e9-95e2-a22f7bb3f270.png)